coalib/core/Core.py

```diff
@@ -0,0 +1,167 @@
+import asyncio
+import concurrent.futures
+import functools
+import logging
+import multiprocessing
+
+
+def get_cpu_count():
```

Comment:
hm it's still functionality someone can use, imo it doesn't have to be "private"